scss/_pagination.scss

```diff
@@ -27,6 +27,13 @@
     outline: $pagination-focus-outline;
     box-shadow: $pagination-focus-box-shadow;
   }
+  
+  &.active {
```

Comment:
This leads to duplicate code. Could you merge both declarations, eg. using `.active, .active > &` as a selector — and then drop the other block in `.page-item`?

Reply:
You are right. I didn't think about it. I have update code.